### DISCUSSION TITLE
How to use my own CUDA code in mmdetection3D?

### DISCUSSION THREAD

**Discussion by Agito555** [open-mmlab/mmdetection3d#2753] [Q&A]
I'm a newbie, or perhaps this is a very basic question. I want to use my own CUDA code in mmdetection3D, but I'm not sure how to do it. Any guidance would be greatly appreciated.

**Comment by Xiangxu-0103** [comment]
Sorry for late reply, currently we move all `CUDA ops`  implemented in `mmcv`. If you want to customize your own CUDA ops, you may refer to [this](https://github.com/open-mmlab/mmdetection3d/tree/main/projects/BEVFusion/bevfusion/ops) for reference.
